conf/broker.conf

```diff
@@ -1144,6 +1144,10 @@ defaultNamespaceBundleSplitAlgorithm=range_equally_divide
 # load shedding strategy, support OverloadShedder and ThresholdShedder, default is ThresholdShedder since 2.10.0
 loadBalancerLoadSheddingStrategy=org.apache.pulsar.broker.loadbalance.impl.ThresholdShedder
 
+# load balance load manager strategy, support LeastLongTermMessageRate and ThresholdLoadManagerStrategy,
+# default is LeastLongTermMessageRate
+loadBalancerLoadManagerStrategy=org.apache.pulsar.broker.loadbalance.impl.LeastLongTermMessageRate
```

Comment:
I wonder if it is better to let the user only configure the classname without the package name, this way we will prevent users to load non-standard classes.
```
loadBalancerLoadManagerStrategy=LeastLongTermMessageRate
loadBalancerLoadManagerStrategy= ThresholdLoadManagerStrategy
```

That would be to add a new API/public extensions point, like Protocol Handlers (and in this case we will require a PIP and a deeper discussion)

Reply:
> I wonder if it is better to let the user only configure the classname without the package name, this way we will prevent users to load non-standard classes.

+1, good point

Reply:
+1

Reply:
Sounds good. I use the full class name because I see other similar logic such as `loadManagerClassName`, `loadBalancerLoadSheddingStrategy`.

Reply:
Updated.